dashboards/codecommit/codecommit_repository_report_age.sp

```diff
@@ -1,4 +1,4 @@
-dashboard "codecommit_repository_age_report" {
+dashboard "codecommit_repository_report_age" {
```

Comment:
@Priyanka-Chatterjee-2000 the name here was correct I just asked you to update the file name